lib/java/src/test/java/org/apache/thrift/transport/TestTByteBuffer.java

```diff
@@ -39,4 +41,29 @@ public void testOverflow() throws Exception {
             () -> byteBuffer.write("Hello World".getBytes(StandardCharsets.UTF_8)));
     assertEquals("Not enough room in output buffer", e.getMessage());
   }
+
+  @Test
+  public void testSmallTConfiguration() throws Exception {
+    // Test that TByteBuffer init fail with small max message size.
+    final TConfiguration configSmall =
+        new TConfiguration(
+            4, TConfiguration.DEFAULT_MAX_FRAME_SIZE, TConfiguration.DEFAULT_RECURSION_DEPTH);
```

Comment:
I'm not sure if we have a java style guide or something but this style looks weird to me, but that might be just me (as I don't write java nowadays).

Reply:
I think there is checkstyle built in the gradle script. This pass `gradle :spotlessApply`.